### DISCUSSION TITLE
Disable/exclude personal PR reminders for a specific team

### DISCUSSION THREAD

**Discussion by pallamidessi** [community/community#53605] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Maybe we've missed it but we would to know if it was possible to not get personal PR reminders (we're using the real-time one quite a bit) for a specific team

Our use-case is that we have a special team for emergency approval that are (co-)codeowners to every repository for visibility and transparency reason but those individuals now get a slack notification for **every** single PRs in the organisation, which is not ideal as you can imagine 😸 

Is there a workaround that we missed ? If not it would be quick handy for schedule reminder to allow one to exclude teams from the reminders 

Thanks a lot !

**Comment by NJacksonDev** [comment]
The below article provides some useful information re: managing scheduled reminders for PRs. Hopefully this helps!

https://docs.github.com/en/organizations/managing-organization-settings/managing-scheduled-reminders-for-your-organization
